### PR TITLE
Porting to python

### PR DESCRIPTION
**Pull request by KeisukeShima**

### CODE REVIEW

File: .github/workflows/test-action.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
         path: ${{ env.ARTIFACTS_DIR }}
 
     - id: metrics-reporter
-      uses: tier4/ros-metrics-reporter@main
+      uses: tier4/ros-metrics-reporter@feature/refactoring
```
Comment (KeisukeShima) [inline .github/workflows/test-action.yml:59]:
```suggestion
      uses: tier4/ros-metrics-reporter@main
```

File: action.yml [hunk 2]
```diff
@@ -59,70 +59,76 @@ outputs:
...
-        cd codechecker
+        pip3 install jinja2 bs4 pandas plotly
+      shell: bash
+
+    - name: Set timestamp
+      run: |
+        date -u '+%Y%m%d_%H%M%S' > ${GITHUB_WORKSPACE}/timestamp.txt
+      shell: bash
+
+    - name: Run CodeCShecker
```
Comment (kmiya) [inline action.yml:87]: Might be CodeChecker :eye: :eye:

Comment (KeisukeShima) [reply action.yml:87]: Oh...

Comment (KeisukeShima) [reply action.yml:87]: Thanks! Created PR #17

Comment (kmiya) [reply action.yml:87]: :+1:

**Pull request merged**